### PR TITLE
Update edl creation script

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Update CreateEDLInstance script to allow for specifying ports.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nericksen** [comment]
Any thoughts on how to get around this validation error in the README?

```
================= Running validation on branch update_EDL_creation_script =================
Validating against origin/master
Running only on staged files
Skipping release notes validation
[ERROR]: Packs/CreateEDLInstance/.pack_ignore: [BA102] - The file type is not supported in the validate command.
The validate command supports: Integrations, Scripts, Playbooks, Incident fields, Incident types, Indicator fields, Indicator types, Objects fields, Object types, Object modules, Images, Release notes, Layouts, Jobs and Descriptions.


================= Running validation on modified files =================

Validating Packs/CreateEDLInstance/README.md as readme
The npm modules: ['@mdx-js/mdx', 'fs-extra', 'commander'] are not installed, Readme mdx validation skipped. Use 'npm install' to install all required node dependencies
[ERROR]: Packs/CreateEDLInstance/README.md: [RM106] - Found the word 'Demisto' in the readme content in lines: [11].

```

**Comment by merit-maita** [comment]
> Any thoughts on how to get around this validation error in the README?
> 
> ```
> ================= Running validation on branch update_EDL_creation_script =================
> Validating against origin/master
> Running only on staged files
> Skipping release notes validation
> [ERROR]: Packs/CreateEDLInstance/.pack_ignore: [BA102] - The file type is not supported in the validate command.
> The validate command supports: Integrations, Scripts, Playbooks, Incident fields, Incident types, Indicator fields, Indicator types, Objects fields, Object types, Object modules, Images, Release notes, Layouts, Jobs and Descriptions.
> 
> 
> ================= Running validation on modified files =================
> 
> Validating Packs/CreateEDLInstance/README.md as readme
> The npm modules: ['@mdx-js/mdx', 'fs-extra', 'commander'] are not installed, Readme mdx validation skipped. Use 'npm install' to install all required node dependencies
> [ERROR]: Packs/CreateEDLInstance/README.md: [RM106] - Found the word 'Demisto' in the readme content in lines: [11].
> ```

about the error in readme, just replace the word demisto with a different word.

**Comment by nericksen** [comment]
hmmm it might be nice to include since the users are a bit unsure that the "Demisto REST API" integration needs to be configured, is the recommendation to remove this statement then?

```
A Demisto REST API integration also needs to be configured for correct operation.
```

**Comment by merit-maita** [comment]
> hmmm it might be nice to include since the users are a bit unsure that the "Demisto REST API" integration needs to be configured, is the recommendation to remove this statement then?
> 
> ```
> A Demisto REST API integration also needs to be configured for correct operation.
> ```

yeah i understand, you can keep it, just replace demisto with demi*sto for example
"demisto" should not be in the readme file

File: Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+##### CreateEDLInstance
+- Update to add Port argument
+
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md:3]: why is it duplicated? its the same as the lower point. isnt it?

File: Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+#### Scripts
+##### CreateEDLInstance
+- Add Port as an argument
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md:6]:
```suggestion
- Added a new argument **Port** to the script, to specify a port to use instead generating a random value
```

**Comment by nericksen** [comment]
ok understood, pushed the change

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.py [hunk 3]
```diff
@@ -643,8 +649,23 @@ def main():
...
+    readable_output = tableToMarkdown("Indicator Feed", output)
+
+    entry = CommandResults(
+        outputs_key_field="Name",
+        outputs_prefix="EDL",
+        outputs=output,
+        readable_output=readable_output,
+        raw_response=results
+    )
+    return_results(entry)
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.py:668]:
```suggestion
    return_results(CommandResults(
        outputs_key_field="Name",
        outputs_prefix="EDL",
        outputs=output,
        readable_output=readable_output,
        raw_response=results
    ))
```

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 args:
 - defaultValue: EDL_PORT_LIST
-  description: The Listname in XSOAR of ports in use
+  description: 'The Listname in XSOAR of ports in use. '
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.yml:3]: why?

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/README.md [hunk 5]
```diff
@@ -6,17 +6,21 @@ Use this automation to create an EDL instance on XSOAR.
...
-| Tags | None  |
+| Tags |  |
 
 ## Inputs
 ---
 
 | **Argument Name** | **Description** |
 | --- | --- |
-| PortListName | The Listname in XSOAR of ports in use |
+| PortListName | The Listname in XSOAR of ports in use.  |
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/Scripts/CreateEDLInstance/README.md:16]: why?

**Review by merit-maita — CHANGES_REQUESTED** [review]
good job!!

File: Packs/CreateEDLInstance/.pack_ignore [hunk 6]
```diff
@@ -0,0 +1,2 @@
+[file:README.md]
+ignore=RM106
```
Comment (merit-maita) [inline Packs/CreateEDLInstance/.pack_ignore:2]:
```suggestion
```
no need to keep it, the error is handled, on the other side it fails the validation

File: Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md (hunk 2 above)
Comment (nericksen) [reply Packs/CreateEDLInstance/ReleaseNotes/1_0_1.md:6]: updated

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.py (hunk 3 above)
Comment (nericksen) [reply Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.py:668]: updated

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.yml (hunk 4 above)
Comment (nericksen) [reply Packs/CreateEDLInstance/Scripts/CreateEDLInstance/CreateEDLInstance.yml:3]: not sure, reverted

File: Packs/CreateEDLInstance/Scripts/CreateEDLInstance/README.md (hunk 5 above)
Comment (nericksen) [reply Packs/CreateEDLInstance/Scripts/CreateEDLInstance/README.md:16]: not sure, so reverted.

File: Packs/CreateEDLInstance/.pack_ignore (hunk 6 above)
Comment (nericksen) [reply Packs/CreateEDLInstance/.pack_ignore:2]: removed

**Review by merit-maita — APPROVED** [review]

**Pull request merged**